src/typography/Typography.module.css

```
@@ -25,3 +25,19 @@
 .mono {
   font-family: var(--mono-font-family);
 }
+
+.thin {
+  font-weight: var(--font-weight-thin);
+}
+
+.normal {
+  font-weight: var(--font-weight-normal);
+}
+
+.bold {
+  font-weight: var(--font-weight-bold);
+}
+
+.extra-bold {
+  font-weight: var(--font-weight-extraBold);
```

This needs to be dashcase 
```suggestion
  font-weight: var(--font-weight-extra-bold);
```

---

Why? We have `extraBold` property. It could be breaking change if we will rename it

---

https://github.com/reaviz/reablocks/blob/master/src/utils/Theme/utils.ts#L7